packages/contracts/mud.config.ts

```diff
@@ -53,15 +53,26 @@ export default mudConfig({
     },
     ActiveGuise: entityRelation,
     GuisePrototype: {
-     ...entityKey,
-     schema: arrayPStat,
+      ...entityKey,
+      schema: arrayPStat,
+    },
+    RNGPrecommit: {
+      ...entityKey,
+      schema: "uint256",
     },
+    // requestId => ownerEntity
+    RNGRequestOwner: entityRelation,
   },
   modules: [
     {
       name: "KeysInTableModule",
       root: true,
       args: [resolveTableId("Experience")],
     },
+    {
+      name: "UniqueEntityModule",
+      root: true,
+      args: [resolveTableId("RNGPrecommit")],
```

Comment:
```suggestion
      args: [],
```
you don't need an argument here, this module isn't specific to any table, and only needs to be added once
(I saw this in some other PR too)